docs/suh-template/analyze/20260525_865_버그_스플래시_화면_백엔드_무응답_시_무한로딩.md

```
@@ -0,0 +1,214 @@
+# 분석 보고서: 스플래시 화면 백엔드 무응답 시 무한 로딩
+
+> 이슈: [#865 ❗[버그][스플래시] 백엔드 서버 무응답 시 스플래시 화면 무한 로딩](https://github.com/TEAM-ROMROM/RomRom-FE/issues/865)  
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "스플래시"

(MD052, reference-links-images)